src/main/resources/CompetitionRequests.sql

```diff
@@ -0,0 +1,52 @@
+---- Вывод итоговой таблицы (общий протокол) со всеми данными
+SELECT
+    d.meters || ' ' || d.name_style AS distance,
+    s.surname,
+    s.first_name,
+    s.date_birth,
+    c.name_city ,
+    p.result_time,
+    comp.name_competitions,
+    p.points
+FROM
+    Protokol p
+JOIN
+    Distance d ON p.id_distance = d.id_distance
+JOIN
+    Sportsmans s ON p.id_sportsman = s.id_sportsman
+JOIN
+    Competitions comp ON p.id_competition = comp.id_competitions
+JOIN
+    City c ON s.city_id = c.id
+ORDER BY result_time
+
+
+--- Топ 3 спортсмена, которые набрали больше всего очков на данных соревнованиях
+SELECT s.surname,s.first_name, p.points
+FROM Protokol p
+JOIN Sportsmans s ON p.id_sportsman = s.id_sportsman
+ORDER BY points DESC
+LIMIT 3;
+
+----Cписок участников на соревнованиях по дистанциям
```

Comment:
No, it is not list of athletes by distance. It is just list of races.  Actually it is your first SELECT without some information like competition and city. Please, read one more time requirements : List of athletes by competition and race(distance). I want to se everyone who seem 50 butterfly on competition 'Swimming Championship'